service/surf/apps/communities/models.py

```diff
@@ -71,8 +71,9 @@ def clean(self):
 
 def validate_logo_size(image):
     width, height = get_image_dimensions(image)
-    if width != 230 or height != 136:
-        raise ValidationError("The proportion of the logo image should be 230x136")
+    # Also support older images of 230x136
+    if (width != 230 and width != 120) or (height != 136 and height != 52):
```

Comment:
Nu kan je ook 230x52 uploaden. Dat is waarschijnlijk niet wat je bedoeld?

Reply:
Jawel, easter egg 🐰

Reply:
Aangepast 😉